Zend/zend_builtin_functions.c

```diff
@@ -2086,10 +2085,7 @@ ZEND_FUNCTION(create_function)
 		if (func->refcount) {
 			(*func->refcount)++;
 		}
-		static_variables = func->static_variables;
-		func->static_variables = NULL;
 		zend_hash_str_del(EG(function_table), LAMBDA_TEMP_FUNCNAME, sizeof(LAMBDA_TEMP_FUNCNAME)-1);
-		func->static_variables = static_variables;
```

Comment:
Was fine in an earlier iteration of the patch … but now … Not quite (free is after refcount), but it will destroy the contents of the static vars ... Fixing and thanks for catching.